apache/trafficcontrol
# Adds the ability to create a CDN-level notification via TP/TO API

## What does this PR (Pull Request) do?

This feature provides users w/ operations or admin role the ability to add a notification to a cdn which does 2 things:

1. creates a notification in the database for the CDN
2. displays a persistent notification for each CDN in TP as such:

![image](https://user-images.githubusercontent.com/251272/108115991-2aa7b900-7058-11eb-92e4-42187e833bad.png)

3. allows anyone w/ operations or admin role to delete the CDN notification

- [x] This PR is related to #1124 

## Which Traffic Control components are affected by this PR?

- Traffic Ops
- Traffic Portal

## What is the best way to verify this PR?
- Launch TP
- Click on a cdn notification icon in the header of TP to create a notification
- Navigate to different views of TP to ensure that the notification is persistent
- Launch a 2nd instance of TP
- In the 2nd instance, click the same cdn notification icon in the header of TP to delete the notification
- Navigate back to TP instance 1 and wait 30 seconds, notification should disappear.

## The following criteria are ALL met by this PR



## Additional Information
